src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
```

Comment:
The task specified to define an enum for different colors. You should not use String for color field in Ball class, but a type of the defined enum.

Reply:
done